internal/plugins/project-quota.go

```diff
@@ -214,8 +218,7 @@ func (m *assetManagerProjectQuota) getQuotaStatus(assetType db.AssetType, projec
 			return res, nil
 		}
 	}
-	//lint:ignore ST1005 Limes is a proper name
-	return nil, fmt.Errorf("Limes does not report %s/%s quota for project %s",
+	return nil, fmt.Errorf("%s/%s quota for project %s isn't reported by Limes",
```

Comment:
Contractions are usually avoided in writing.
```suggestion
	return nil, fmt.Errorf("%s/%s quota for project %s is not reported by Limes",
```